rollup.config.js

```diff
@@ -0,0 +1,108 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import postcss from 'rollup-plugin-postcss';
+import builtins from 'rollup-plugin-node-builtins';
+import json from 'rollup-plugin-json';
+import babel from 'rollup-plugin-babel';
+import postcssImport from 'postcss-import';
+import postcssPresetEnv from 'postcss-preset-env';
+import postcssReporter from 'postcss-reporter';
+import execute from 'rollup-plugin-execute';
+import cleaner from 'rollup-plugin-cleaner';
+import postcssCustomProperties from 'postcss-custom-properties';
+import postcssCustomMediaQueries from 'postcss-custom-media';
+import postcssPostcssColorModFunction from 'postcss-color-mod-function';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import svgrPlugin from '@svgr/rollup';
+import babelOptions from '@commercetools-frontend/babel-preset-mc-app';
+import pkg from './package.json';
+
+const browserslist = {
+  production: ['>1%', 'not op_mini all', 'ie 11'],
+};
+
+const postcssPlugins = [
+  postcssImport(),
+  postcssPresetEnv({
+    browsers: browserslist.production,
+    autoprefixer: { grid: true },
+  }),
+  postcssCustomProperties({
+    preserve: false,
+  }),
+  postcssCustomMediaQueries(),
+  postcssPostcssColorModFunction(),
+  postcssReporter(),
+];
+
+export default [
+  {
+    input: 'src/index.js',
+    output: [
+      {
+        file: `dist/${pkg.module}`,
+        format: 'esm',
+      },
+      {
+        file: `dist/${pkg.main}`,
+        format: 'cjs',
+      },
+    ],
+    plugins: [
+      cleaner({
+        silent: true,
+        targets: ['./dist/'],
+      }),
+      peerDepsExternal(),
+      builtins(),
+      commonjs({
+        include: 'node_modules/**',
+        namedExports: {
+          'node_modules/react-is/index.js': ['isValidElementType'],
+          'node_modules/flatpickr/dist/l10n/de.js': ['German'],
+        },
```

Comment:
Can you explain why we need this list and in which situation we should put stuff in it?

Reply:
@emmenko Yeah a good place to start is [here](https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports). There's also a good discussion [here](https://github.com/rollup/rollup-plugin-commonjs/issues/211) about why we need it in rollup but don't in webpack.

TL:DR = 

```
Both options are valid, but that's the tradeoff to be aware of — more config (Rollup, when using modules like React), or larger bundles (webpack).

```

Reply:
Ok I see. So I guess you will get an error at some point so you know that you have to patch it, right?

Reply:
Exactly! It seems quite natural that some modules just can't be converted from commonjs and this is a valid fallback. When you observe the error after building, this is the place to add the file to.